Documentation/api-reference.md

```diff
@@ -166,6 +166,56 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
   end sub, context)
 ```
 
+#### Example of SendEvent with IdentityMap:
+
+> **Note**
+> `sendEvent` API allows passing custom identityMap with custom identifiers to the edge network. Create the map using identifier namespace and configure the "primary" and "authenticatedState" or the individual identity as per the application requirements.
```

Comment:
- to the Edge Network
- let's remove the note here and add it as a description under this subtitle
- some typos & rewording: Create the map using the identifier namespace as key and pass in the identity items for the namespace as an array. Configure the "primary" and "authenticatedState" per individual identity item per your application's requirements.